app/package.json

```
@@ -38,7 +38,8 @@
     "redux-persist": "6.0.0",
     "redux-saga": "1.1.3",
     "redux-sagas-injector": "1.1.1",
-    "selectorator": "4.0.3"
+    "selectorator": "4.0.3",
+    "todoist-rest-api": "^1.3.4"
```

Please use exact package versions to make sure that all instances (development, production, CI) use exactly the same versions:

```suggestion
    "todoist-rest-api": "1.3.4"
```

With yarn, you can enforce this with `yarn add package-name --exact` (or `-E`), with npm it's `npm install package-name --save --save-exact`.